doc/cargo_command_execution.md

```diff
@@ -0,0 +1,186 @@
+# Cargo Command Execution Page
+
+The extension allows a developer to execute any of built-in cargo commands.
+
+These commands are:
+
+* bench
+* build
+* check
+* clean
+* clippy
+* doc
+* new
+* run
+* test
+* update
+
+These commands available through the command palette (CTRL+P).
+
+These commands have prefix `"Cargo: "`.
+
+## Execute Command On Save
+
+The extension supports executing a command after saving the document opened in the active text document.
+
+The `"rust.actionOnSave"` configuration parameter specifies a command to execute.
+
+The possible values:
+
+* `"build"` - the extension executes `"Cargo: Build"`
+* `"check"` - the extension executes `"Cargo: Check"`
+* `"clippy"` - the extension executes `"Cargo: Clippy"`
+* `"run"` - the extension executes `"Cargo: Run"`
+* `"test"` - the extension executes `"Cargo: Test"`
+* `null` - the extension does nothing
+
+By default, it is `null`.
+
+## Current working directory determination
```

Comment:
FYI some markdown renderers need to have an empty line between a headline and the next paragraph so this may not render correctly in some places